js/script.js

```diff
@@ -0,0 +1,87 @@
+// Интерактивная карта по клику
+
+var maplink = document.querySelector(".contacts-and-map__image");
+
+if (maplink) {
+  var mappopup = document.querySelector(".popup-map");
+  var mapclose = mappopup.querySelector(".form-close-button");
+
+  maplink.addEventListener("click", function(evt) {
+    evt.preventDefault();
+    mappopup.classList.add("modal-show");
+  });
+
+  mapclose.addEventListener("click", function(evt) {
+    evt.preventDefault();
+    mappopup.classList.remove("modal-show");
+  });
+
+  window.addEventListener("keydown", function(evt) {
+    if (evt.keyCode === 27) {
+      evt.preventDefault();
+      if (mappopup.classList.contains("modal-show")) {
+        mappopup.classList.remove("modal-show");
+      }
+    }
+  });
+}
+
+// Корзина в каталоге
+var buyButton = document.querySelectorAll(".catalog-goods-list__buy-button");
+var cart = document.querySelector(".cart");
+var buyClose = cart.querySelector(".form-close-button");
+var buyContinue = cart.querySelector(".cart__button-continue")
```

Comment:
```suggestion
var buyContinue = cart.querySelector(".cart__button-continue");
```

проверь все точки с запятыми, еще есть места, где они не стоят у тебя